{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2015-0222",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "ModelMultipleChoiceField in Django 1.6.x before 1.6.10 and 1.7.x before 1.7.3, when show_hidden_initial is set to True, allows remote attackers to cause a denial of service by submitting duplicate values, which triggers a large number of SQL queries.",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2015-0222",
				"url": "https://www.suse.com/security/cve/CVE-2015-0222"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 913055 for CVE-2015-0222",
				"url": "https://bugzilla.suse.com/913055"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 919763 for CVE-2015-0222",
				"url": "https://bugzilla.suse.com/919763"
			},
			{
				"category": "external",
				"summary": "Advisory link for SUSE-SU-2015:0563-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-March/001310.html"
			},
			{
				"category": "external",
				"summary": "Advisory link for SUSE-SU-2015:1109-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001452.html"
			},
			{
				"category": "external",
				"summary": "Advisory link for SUSE-SU-2015:1112-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001453.html"
			}
		],
		"title": "SUSE CVE CVE-2015-0222",
		"tracking": {
			"current_release_date": "2023-12-08T04:47:29Z",
			"generator": {
				"date": "2023-02-15T05:23:52Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2015-0222",
			"initial_release_date": "2023-02-15T05:23:52Z",
			"revision_history": [
				{
					"date": "2023-02-15T05:23:52Z",
					"number": "2",
					"summary": "Current version"
				},
				{
					"date": "2023-12-08T04:47:29Z",
					"number": "3",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "3"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Enterprise Storage 1.0",
								"product": {
									"name": "SUSE Enterprise Storage 1.0",
									"product_id": "SUSE Enterprise Storage 1.0",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:ses:1.0"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 4",
								"product": {
									"name": "SUSE OpenStack Cloud 4",
									"product_id": "SUSE OpenStack Cloud 4",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:cloud:4"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-Django-1.6.11-4.1",
								"product": {
									"name": "python-Django-1.6.11-4.1",
									"product_id": "python-Django-1.6.11-4.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:1.6.11:*:*:*:*:*:*:*"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python-django-1.5.12-0.7.1",
								"product": {
									"name": "python-django-1.5.12-0.7.1",
									"product_id": "python-django-1.5.12-0.7.1",
									"product_identification_helper": {
										"cpe": "cpe:2.3:a:djangoproject:django:1.5.12:*:*:*:*:*:*:*"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django-1.6.11-4.1 as component of SUSE Enterprise Storage 1.0",
					"product_id": "SUSE Enterprise Storage 1.0:python-Django-1.6.11-4.1"
				},
				"product_reference": "python-Django-1.6.11-4.1",
				"relates_to_product_reference": "SUSE Enterprise Storage 1.0"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-django-1.5.12-0.7.1 as component of SUSE OpenStack Cloud 4",
					"product_id": "SUSE OpenStack Cloud 4:python-django-1.5.12-0.7.1"
				},
				"product_reference": "python-django-1.5.12-0.7.1",
				"relates_to_product_reference": "SUSE OpenStack Cloud 4"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2015-0222",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2015-0222"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "ModelMultipleChoiceField in Django 1.6.x before 1.6.10 and 1.7.x before 1.7.3, when show_hidden_initial is set to True, allows remote attackers to cause a denial of service by submitting duplicate values, which triggers a large number of SQL queries.",
					"title": "CVE description"
				}
			],
			"product_status": {},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2015-0222",
					"url": "https://www.suse.com/security/cve/CVE-2015-0222"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 913055 for CVE-2015-0222",
					"url": "https://bugzilla.suse.com/913055"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 919763 for CVE-2015-0222",
					"url": "https://bugzilla.suse.com/919763"
				},
				{
					"category": "external",
					"summary": "Advisory link for SUSE-SU-2015:0563-1",
					"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-March/001310.html"
				},
				{
					"category": "external",
					"summary": "Advisory link for SUSE-SU-2015:1109-1",
					"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001452.html"
				},
				{
					"category": "external",
					"summary": "Advisory link for SUSE-SU-2015:1112-1",
					"url": "https://lists.suse.com/pipermail/sle-security-updates/2015-June/001453.html"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Enterprise Storage 1.0:python-Django-1.6.11-4.1",
						"SUSE OpenStack Cloud 4:python-django-1.5.12-0.7.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2015-01-13T20:16:18Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2015-0222"
		}
	]
}
